cli/medperf/utils.py

```diff
@@ -21,9 +22,69 @@
 import medperf.config as config
 
 
+def parse_context_args(ctx_args: List[str]) -> dict:
+    """Parses extra arguments received from typer into a dictionary of args
+
+    Args:
+        ctx_args (List[str]): List of extra cli arguments
+
+    Returns:
+        dict: dictionary of key-value cli arguments
+    """
+    args = []
+    for arg in ctx_args:
+        args += arg.split("=")
```

Comment:
currently this will work:
`medperf profile set --key=val=--=`
Let's have here an assertion that only one `=` sign is present. (idealism :))

Reply:
I just realized: should be only one `=` sign or no `=` signs.